package.json

```diff
@@ -20,9 +20,15 @@
     "dist/"
   ],
   "main": "index.js",
+  "exports": {
+    "import": "dist/faunadb.es.js",
+    "require": "dist/faunadb.js"
+  },
+  "type": "module",
+  "module": "dist/faunadb.es.js",
```

Comment:
@steida I added the `module` field for future support though [this StackOverflow post](https://stackoverflow.com/a/42817320) says it is just a proposal at this point.

To be safe, I also added the `exports` objects as per these Node docs: https://nodejs.org/api/esm.html#esm_conditional_exports